docs/arch/adr-04-merging-mainstream-to-specialist-topics.md

```diff
@@ -1,38 +1,62 @@
-# Merging of Mainstream Topics into Specialist Topics
+# Merging of Mainstream Browse Topics into Specialist Topics
 
 ## Status
 
-Accepted
+Proposed
 
 ## Context
 
-GOV.UK currently maintains 2 separate topic trees that represent a navigational structure for users. Mainstream Browse is the oldest and considered legacy. Specialist Topics are newer but need tidying up and do not power breadcrumbs.
+GOV.UK currently maintains 3 topic systems that represent a navigational structure for users. The Topic Taxonomy (schema: `taxon`), Specialist Topics (schema: `topic`) and Mainstream Browse (schema: `mainstream_browse_page`).
 
-Previous attempts to merge the two stalled (since 2015).
+Mainstream Browse is the oldest topic system. Documents are tagged to Mainstream Browse topics by GDS Editors in [Content Tagger](https://github.com/alphagov/content-tagger) or [Publisher](https://github.com/alphagov/publisher). Tagging adds a Mainstream Browse breadcrumb. For the tagged content to be visible on the Mainstream Browse topic page, a further curation step is required in collections publisher.
 
-The newest tree, Taxonomy, does not affect navigation and is treated as a purely metadata type categorisation tree and is ignored by this work.
+The Specialist Topic system is concerned with specialist content, and is the only topic system integrated with email notifications. Government publishers are able to tag content to the Specialist Topic system in Whitehall. Doing so adds a Specialist Topic breadcrumb to the page, and results in that content appearing on some Specialist Topic pages.
 
-There is no connection between Mainstream Browse and the Specialist Topics. Users who navigate Mainstream Browse are siloed away from Specialist Topics content. It also causes a confusing and inconsistent way for tagging and curating content.
+The most recent topic system is the Topic Taxonomy, which was intended to replace Specialist Topics and Mainstream Browse. Whilst tagging a piece of content to this system will provide navigational elements if the content is not tagged to other topic systems, the Topic Taxonomy is not viewed as a navigational system. It is primarily concerned with search, and the filtering of search results.
+
+#### Problems
+
+* There is no connection between Mainstream Browse and Specialist Topics, i.e. users who navigate to a Mainstream Browse page are siloed away from Specialist Topic content.
```

Comment:
Is this because Whitehall publishers cannot tag to mainstream browse topics, only to specialist topics?

Also, how is the list of specialist topics maintained?